userland/Configuration.mk

```diff
@@ -34,7 +34,7 @@ TOCK_ARCHS ?= cortex-m0 cortex-m4
 
 # This could be replaced with an installed version of `elf2tbf`
 ELF2TBF ?= cargo run --manifest-path $(abspath $(TOCK_USERLAND_BASE_DIR))/tools/elf2tbf/Cargo.toml --
-ELF2TBF_ARGS += -n $(PACKAGE_NAME)
+ELF2TBF_ARGS += -p -n $(PACKAGE_NAME)
```

Comment:
In the interest of readability and 2017 being the year of "characters are cheap", how do we feel about making the long form of that argument `--include-pic-header` and using the long form here in the Makefile?

(I'm cool with leaving `-p` as an alias for quick cli testing)